modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/config/types.scala

```diff
@@ -78,15 +78,34 @@ object types {
   case class Weight(value: NonNegLong)
 
   case class ProgramsDistributionConfig(
-    weights: Map[Address, Weight],
-    remainingWeight: Weight
+    weights: Map[Address, NonNegFraction],
+    validatorsWeight: NonNegFraction,
+    delegatorsWeight: NonNegFraction
   )
 
   case class OneTimeReward(epoch: EpochProgress, address: Address, amount: TransactionAmount)
 
+  /** Configuration for the emission formula parameters
+    */
+  case class EmissionConfig(
+    epochsPerYear: PosLong, // Number of epochs in a year (yardstick for time-based calculations)
+    yearInitial: EpochProgress, // Initial reference year epoch
+    iTarget: NonNegFraction, // Target inflation rate (0.5%)
+    iInitial: NonNegFraction, // Initial inflation rate (6%)
+    lambda: NonNegFraction, // Decay constant (0.1)
+    dagPrices: SortedMap[EpochProgress, NonNegFraction], // DAG price data - to be replaced by oracle in the future
+    iImpact: NonNegFraction, // Impact factor (0.35)
+    secondsPerEpoch: PosLong, // Seconds per epoch scaling factor (empirically measured as 43 seconds)
```

Comment:
I only see one reference to this value, and the reference has no usages beyond an initial val assignment. I think we can remove this -- epochsPerYear should account for the 43 second average as part of the yardstick definition, so I'm not sure if this is an artifact from an earlier branch.